.github/workflows/release.yml

```
@@ -3,35 +3,114 @@ name: Release
     strategy:
       matrix:
-        os:
-          - ubuntu-latest
-          - macos-latest
-          # - windows-latest
+        os: [ubuntu-22.04, macos-13, windows-2022]
     runs-on: ${{ matrix.os }}
+    env:
+      BIN_NAME: gold_digger
+    outputs:
+      hashes: ${{ steps.hash.outputs.hashes }}
```

actionlint (1.7.7)

property "hash" is not defined in object type {}

(expression)